dietpi/dietpi-software

```diff
@@ -11590,6 +11622,30 @@ _EOF_
 
 		fi
 
+		#Docker Config
+		INSTALLING_INDEX=162
+		if (( ${aSOFTWARE_INSTALL_STATE[$INSTALLING_INDEX]} == 1 )); then
+
+			#Create docker user and add to the docker group
+			#adduser --system docker-user
+			#usermod -aG docker docker-user
+
+			# Create directory for docker containers
+			mkdir /mnt/dietpi_userdata/docker-data
+
+			# stop service
+			systemctl stop docker.service
+
+			# Set container(s) locations in /lib/systemd/system/docker.service
+			sed -i '/ExecStart=\/usr\/bin\/dockerd -H fd:\/\// c\ExecStart=\/usr\/bin\/dockerd -g \/mnt\/dietpi_userdata\/docker-data -H fd:\/\/' /lib/systemd/system/docker.service
+
+			# Add to service for dietpi
+			cat << _EOF_ >> /DietPi/dietpi/.dietpi-services_include
```

Comment:
I'd prefer if we add ```'docker'``` to ```dietpi-services```:
https://github.com/Fourdee/DietPi/blob/master/dietpi/dietpi-services#L105

We do this for all Software services available in ```dietpi-software```.

Reply:
I had no idea it was that easy great.